src/pages/guides/Webhooks/index.md

```diff
@@ -0,0 +1,195 @@
+|`file.upload.completed`	|A File has been **uploaded**	|
+|`file.versioned`	|A File version has been created	|
+
+**Folders**
+
+|Event	|Description	|
+|---	|---	|
+|`folder.created`	|A new Folder has been **created**	|
+|`folder.updated`	|A Folder's settings are **updated**	|
+|`folder.deleted`	|A Folder has been **deleted**	|
+
+### Comments
+
+|Event	|Description	|
+|---	|---	|
+|`comment.created`	|A new Comment or Reply has been **created**	|
+|`comment.updated`	|A Comment has been updated	|
+|`comment.deleted`	|A Comment was **deleted**	|
+|`comment.completed`	|A Comment was marked as **completed**	|
+|`comment.uncompleted`	|A Comment was marked as **uncompleted**	|
+
+### Metadata
+
+|Event	|Description	|
+|---	|---	|
+|`metadata.value.updated`	|Metadata field on an Asset updated	|
+
+## Security
+
+By default, all webhooks have a signing key. This non-configurable signing secret can be used to verify that the request originates from Frame.io. The response payload for the webhook you’ve configured includes the signing secret specific to this webhook. This secret is **only provided in this initial webhook create response**, so store it somewhere safe in your secrets storage or environment variables.  Use it later to verify the webhook is coming directly from our servers and was not intercepted or manipulated in any way.
+
+### Verifying Webhook Signatures
+
+To guard an integration against man-in-the-middle and replay attacks it is essential to verify the signature of the webhook payload. Verification ensures that webhook payloads were actually sent by Frame.io and payload content has not been modified in transport.
+
+Included with the `POST` request are the following HTTP headers:
+
+|Header Name	|Description	|Example	|
+|---	|---	|---	|
+|`X-Frameio-Request-Timestamp`	|The timestamp the request was sent	|`1604004499`	|
+|`X-Frameio-Signature`	|The compute webhook signature	|`v0=a77ce6856e609c884575c2fd211d07a9ad1c3f72e19c06ff710e8f086ffca883`	|
+|`user-agent: "Frame.io V4 API"`	|User agent in the header for v4	|	|
+|`user-agent: "Frame.io Legacy API"`	|User agent in the header for v3	|	|
+
+* **The timestamp** is the system time from Frame.io's systems when the outbound webhook is sent. This can be used to prevent [replay](https://en.wikipedia.org/wiki/Replay_attack) attacks. We recommended verifying this time is within 5 minutes of local time.
+
+* **The signature** is a HMAC SHA256 hash using the signing key provided when the Webhook is first created.
+
+**Follow these steps to verify the signature:**
+
+1. Extract the signature from the HTTP headers.
+2. Create a message to sign by combining the version, delivery time, and request body: `v0:timestamp:body.`
+3. Compute the HMAC SHA256 signature using your signing secret.
+    1. Note: The provided signature is prefixed with `v0=`. Currently Frame.io only has this one version for signing requests. Be sure this prefix is prepended to your computed signature.
+4. Compare!
+
```

Comment:
We had code samples for verifying the webhook signature on the legacy API docs, we should probably surface the same samples here and use the multi-code block element in the MDX docs (need to change the extension from .md to .mdx).